docs/source/examples/simple-landscape-braket.md

```diff
@@ -0,0 +1,203 @@
+
+    Returns:
+        The expectation value of the ZZ observable.
+    """
+    device = LocalSimulator('braket_dm')
+    # Evaluate the ZZ expectation value
+    circ.expectation(observable=Observable.Z() @ Observable.Z(), target=range(2))
+    
+    task = device.run(circ, shots = 0)
+    result = task.result()
+    return result.values
+
+# Strength of noise channel
+noise_level = 0.04
+
+def executor_with_noise(circ: Circuit) -> float:
+    """Simulates the execution of a circuit with depolarizing noise.
+
+    Args:
+        circ: The input circuit.
+
+    Returns:
+        The expectation value of the ZZ observable.
+    """
+    # Add depolarizing noise to the circuit
+    noise = Noise.Depolarizing(probability=noise_level)
+    circ.apply_gate_noise(noise)
+    # Use the noiseless_executor function to return the expectation value of the ZZ observable for the noisy circuit
+    return noiseless_executor(circ)
+```
+
+```{note}
+The above code block uses depolarizing noise, but any Braket `Noise` channel can be substituted in.
+```
+
+## Computing the landscape without noise
+
+We now compute the energy landscape $\langle H \rangle(\gamma) =\langle Z \otimes Z \rangle(\gamma)$ on the noiseless simulator.
+
+```{note}
+The remaining code in this tutorial is generic and does not depend on a particular frontend.
+```
+
+```{code-cell} ipython3
+gammas = np.linspace(0, 2 * np.pi, 50)
+noiseless_expectations = [noiseless_executor(variational_circuit(g)) for g in gammas]
+```
+
+The following code plots the values for visualization.
+
+```{code-cell} ipython3
+plt.figure(figsize=(8, 6))
+plt.plot(gammas, noiseless_expectations, color="g", linewidth=3, label="Noiseless")
+plt.title("Energy landscape", fontsize=16)
+plt.xlabel(r"Ansatz angle $\gamma$", fontsize=16)
+plt.ylabel(r"$\langle H \rangle(\gamma)$", fontsize=16)
+plt.legend(fontsize=14)
+plt.ylim(-1, 1);
+plt.show()
+```
+
+## Computing the unmitigated landscape
+We now compute the unmitigated energy landscape $\langle H \rangle(\gamma) =\langle Z \otimes Z \rangle(\gamma)$
+in the following code block.
+
+```{code-cell} ipython3
+gammas = np.linspace(0, 2 * np.pi, 50)
+expectations = [executor_with_noise(variational_circuit(g)) for g in gammas]
+```
+
+The following code plots these values for visualization along with the noiseless landscape.
+
+```{code-cell} ipython3
+plt.figure(figsize=(8, 6))
+plt.plot(gammas, noiseless_expectations, color="g", linewidth=3, label="Noiseless")
+plt.scatter(gammas, expectations, color="r", label="Unmitigated")
+plt.title(rf"Energy landscape", fontsize=16)
+plt.xlabel(r"Ansatz angle $\gamma$", fontsize=16)
+plt.ylabel(r"$\langle H \rangle(\gamma)$", fontsize=16)
+plt.legend(fontsize=14)
+plt.ylim(-1, 1);
+plt.show()
+```
+
+## Computing the mitigated landscape
+We now repeat the same task but use Mitiq to mitigate errors.
+We initialize a RichardsonFactory with scale factors `[1, 3, 5]` and we get a mitigated executor as follows.
```

Comment:
Adding a link to the RichardsonFactory entry in the API-doc here would be good: https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.zne.inference.RichardsonFactory